frontend/src/views/course/index.vue

```diff
@@ -1,27 +1,36 @@
 <template>
-  <div class="main-content course">
+  <div class="main-content">
     <AppBreadCrumb/>
-    <div class="course-header">
-        <div class="course-header-title">
-            <h2>{{ courseInfo.code }}</h2>
-            <h3>{{ courseInfo.name }}</h3>
-            <CourseLinks   :handbookURL="courseInfo.handbookURL" :outlineURL="courseInfo.outlineURL"/>
-            <CourseRatings :ratings="courseRatings"/>
-        </div>
+    <div class="course">
+        <div class="course-header">
+            <div class="course-header-title">
+                <div class="key-data">
+                    <div class="left">
+                        <h2>{{ courseInfo.code }}</h2>
+                        <h3>{{ courseInfo.name }}</h3>
+                    </div>
+                    <div class="right">
+                        <OverallRating :rating="courseRatings[0]"/>
+                    </div>
+                </div>
 
-        <router-link :to="{name: 'info'}">
-            <TabButton :active="this.$route.name == 'info'">Info</TabButton>
-        </router-link>
+                <CourseLinks   :handbookURL="courseInfo.handbookURL" :outlineURL="courseInfo.outlineURL"/>
+                <CourseRatings :ratings="courseRatings"/>
+            </div>
+            <router-link :to="{name: 'info'}">
+                <TabButton :active="this.$route.name == 'info'">Reviews</TabButton>
+            </router-link>
 
-        <router-link :to="{name: 'questions'}">
-            <TabButton :active="this.$route.name == 'questions'">Questions</TabButton>
-        </router-link>
+            <router-link :to="{name: 'questions'}">
+                <TabButton :active="this.$route.name == 'questions'">Questions</TabButton>
+            </router-link>
 
-        <router-link :to="{name: 'reviews'}">
-            <TabButton :active="this.$route.name == 'reviews'">Reviews</TabButton>
-        </router-link>
-    </div>
+        </div>
 
+        <div class="course-info">
```

Comment:
I know it's not finished, but I just don't think this should be on the side. Plz put it back for now I think?

Reply:
So remove entirely?

Reply:
yeeah